misago/users/management/commands/removeips.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandError
+
+from misago.conf import settings
+from misago.core.utils import ANONYMOUS_IP
+from misago.users.signals import anonymize_old_ips
+
+class Command(BaseCommand):
+    help =  "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."
+    
+    def handle(self, *args, **options):
+      if not settings.MISAGO_IP_STORE_TIME:
+        print("IP anonymization is disabled!")
```

Comment:
Don't use `print()` in management commands without *very good reason*. `self.stdout.write()` is much better.